kik_unofficial/client.py

```diff
@@ -573,17 +573,17 @@ def change_email(self, old_email, new_email):
         self.logger.info(f"Changing account email to '{new_email}'")
         return self._send_xmpp_element(account.ChangeEmailRequest(self.password, old_email, new_email))
 
-    def disconnect(self, close=False):
+    def disconnect(self, exiting=False):
         """
         Closes the connection to kik's servers.
         """
         self.logger.warning("Disconnecting.")
         self.connection.close()
         self.is_expecting_connection_reset = True
 
-        if close:
+        if exiting:
```

Comment:
And let's add a comment that explains what happens in this case

Reply:
I have changed this to is_exiting and added a comment and doc string